examples/grpc-route.yaml

```diff
@@ -1,14 +1,17 @@
-apiVersion: gateway.networking.k8s.io/v1beta1
-kind: HTTPRoute
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: GRPCRoute
 metadata:
   name: grpc
```

Comment:
Could you change this name to some thing like my-grpc-route?

Reply:
Why?